bd-runtime/src/runtime.rs

```diff
@@ -868,8 +868,9 @@ pub mod stats {
     5.minutes()
   );
 
-  // This controls how many unique counters we allow before rejecting new metrics. This limit
-  // prevents unbounded growth of metrics, which could result in the system running out of memory.
+  // This controls how much tag cardinality we allow before rejecting new metrics *per workflow*.
+  // This limit prevents unbounded growth of metrics, which could result in the system running out
+  // of memory.
```

Comment:
Do we still want a global limit? Not sure if it's worth it.

Reply:
doesn't seem super important